components/v1/Navigation.tsx

```diff
@@ -41,7 +48,7 @@ const Navigation: FunctionComponent<NavigationProps> = ({ marketsActive, portfol
                     </Link>
                 </div>
                 <div className="w-3/5 flex-grow justify-center space-x-4 text-center sm:w-fit sm:space-x-8 sm:text-left">
-                    <Link href="/arbitrum/markets">
+                    <Link href={router.pathname.includes("binance") ? "/markets/binance" : "/markets/arbitrum"}>
```

Comment:
it's this thing works when you're on portfolio pages ?

Reply:
For now it will redirect user to `/markets/arbitrum` when they press the `markets button` in navigation. But I'm planning to fix it on the next PR (when handling the new portfolio page)